project/sites/src/themes/src_theme/src/scss/components.scss

```diff
@@ -4,3 +4,19 @@
  */
 @import 'init/index';
 @import '../../node_modules/nicsdru_unity_theme/src/scss/3_components/index';
+
+#logo {
```

Comment:
This code has now moved to the unity theme so it's no longer needed in sub themes. May need to run lando composer install to get the latest.

Reply:
Thanks, I have updated this code, that should be it now